foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -105,6 +108,41 @@ public OpenAiChatCompletionOutput chatCompletion(
     return execute("/chat/completions", parameters, OpenAiChatCompletionOutput.class);
   }
 
+  /**
+   * Generate a completion for the given prompt.
+   *
+   * @param parameters the prompt, including messages and other parameters.
+   * @return A stream of chat completions deltas
+   * @throws OpenAiClientException if the request fails
+   */
+  @Nonnull
+  public Stream<OpenAiChatCompletionDelta> streamChatCompletion(
+      @Nonnull final OpenAiChatCompletionParameters parameters) throws OpenAiClientException {
+    parameters.setStream(true);
+    return streamChatCompletion("/chat/completions", parameters, OpenAiChatCompletionDelta.class);
+  }
+
+  /**
+   * Generate a completion for the given prompt.
```

Comment:
(minor) The javadoc should explain a bit how this is different form `streamChatCompletion`. Also, this simpler variant currently this is not mentioned in the docs.

Reply:
Mentioned in the docs